spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -271,6 +271,8 @@ def get_bgcolor(self, index):
         if not self.bgcolor_enabled:
             return
         value = self.get_value(index.row(), column)
+        if np.isnan(value):
+            return Qt.white

```

Comment:
However, this isn't really necessary at all, since you can just add the previous line as a condition to the if statement directly below, which does exactly what you want anyway.